src/cmd/build.rs

```diff
@@ -391,21 +392,34 @@ fn do_optimization(
     optimization_level: OptimizationPasses,
 ) -> Result<()> {
     // check `wasm-opt` is installed
-    if which::which("wasm-opt").is_err() {
+    let which = which::which("wasm-opt");
+    if which.is_err() {
         anyhow::bail!(
-            "{}",
-            "wasm-opt is not installed. Install this tool on your system in order to \n\
-             reduce the size of your contract's Wasm binary. \n\
-             See https://github.com/WebAssembly/binaryen#tools"
-                .bright_yellow()
+            "wasm-opt not found!\n\
```

Comment:
We might add that we expect it to be installed on the `PATH`, to be absolutely clear.